ga_resources/tasks.py

```diff
@@ -7,7 +7,7 @@
 from django.utils.timezone import utc
```

Comment:
Can we put these tasks in hs_core/tasks.py instead?  It would be nice to get rid of ga_resources in the near future.